run.sh

```diff
@@ -3,12 +3,16 @@
 set -x
 set -e
 
-ARGS=(--name vertx-fib-demo -p 8080:8080 -p 9093:9093)
+if [ -z "$HTTP_PORT" ]; then
+    HTTP_PORT=8080
+fi
+
+ARGS=(--name vertx-fib-demo -p $HTTP_PORT -p 9093:9093)
 
 if podman pod exists container-jfr; then
     ARGS+=(--pod container-jfr)
 fi
 
 ARGS+=(--rm -it quay.io/andrewazores/vertx-fib-demo:latest)
 
-podman run "${ARGS[@]}"
+podman run --env HTTP_PORT="$HTTP_PORT" "${ARGS[@]}"
```

Comment:
For extra style points, instead of putting the `--env` directly on this line, you could do `ARGS+=` to append it to the array just above the previous one where `--rm -it` are added. This makes no actual behaviour change, it's purely stylistic.